tracing/src/lib.rs

```diff
@@ -421,6 +421,7 @@
 //! // - "message", with the value "the answer to the ultimate question of life, the
 //! //    universe, and everything is 42."
 //! event!(
+//!     name: "life",
```

Comment:
this example has a block comment explaining the span that will be recorded, and it doesn't mention the overridden name: 
https://github.com/tokio-rs/tracing/pull/2699/files#diff-ea2f30245316830135d5b43541a7be0930b4bfded454b5396509770c39a59c7cR417-R422

can we either update that comment, or remove the overridden name from this example?

my preference would probably be to add *new* examples for overriding event names, rather than including them in existing examples. this example is intended to demonstrate the way that the message field is generated, and overriding the name is not related to message generation. my preference would be to not include things not directly related to the specific thing being demonstrated, so that the user doesn't think they're related...

Reply:
Yeah, makes sense.

Reply:
great, thanks!

Reply:
Ok, addressed!